### DISCUSSION TITLE
NPM Error

### DISCUSSION THREAD

**Discussion by Rushil20** [community/community#47236] [Other Feature Feedback, Questions, & Ideas] [answered]
### #47236 

Question

### Body

Hello Everyone,

I'm trying to make a react project but getting same error with different types of npm. I'm attaching the picture of the error, Thank you in advance for your suggestions and help.

![image](https://user-images.githubusercontent.com/42838884/218453945-6bfeb52a-3b0f-465b-87cc-95a1d6043be5.png)

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @Rushil20 

There is a discussion forum for npm itself, I believe it is easier to get help.

https://github.com/npm/feedback/discussions

**Reply by Rushil20** [reply]
Thanks for suggestion.
